src/containers/Tenant/Diagnostics/Partitions/Partitions.tsx

```diff
@@ -43,215 +40,162 @@ export const b = block('ydb-diagnostics-partitions');
 
 interface PartitionsProps {
     path?: string;
-    type?: EPathType;
-    nodes?: Record<number, string>;
-    consumers?: string[];
 }
 
-export const Partitions = ({path, type, nodes, consumers}: PartitionsProps) => {
-    const isCdcStream = isCdcStreamEntityType(type);
-
+export const Partitions = ({path}: PartitionsProps) => {
     const dispatch = useDispatch();
 
-    const [generalSearchValue, setGeneralSearchValue] = useState('');
-    const [partitionIdSearchValue, setPartitionIdSearchValue] = useState('');
-
+    // Manual path control to ensure it updates with state reset
+    // so no request with wrong params will be sent
     const [componentCurrentPath, setComponentCurrentPath] = useState(path);
```

Comment:
When new `path` arrives, it triggers component's re-render. What do we need to ensure?

Reply:
No, this way it won’t render correctly on autorefresh. Also first render could happen with already loaded topic data (when navigating from info or consumers tabs)